packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -72,6 +76,8 @@ class ${class_name}:
         if grpc_channel is not None:
             self._stub = v2_measurement_service_pb2_grpc.MeasurementServiceStub(grpc_channel)
         self._create_file_descriptor()
+        self._pin_map_path = ""
+        self._sites = [0]
```

Comment:
Can we expose the sites parameter as a public attribute and set its default value to 0? This would give users the flexibility to specify the sites from the client side.